docs/core/tools/dotnet-publish.md

```diff
@@ -52,7 +52,7 @@ The `dotnet publish` command's output is ready for deployment to a hosting syste
 
 `PROJECT`
 
-The project to publish. If not specified, it defaults to the current directory.
+The project to publish. It's either a path to a [csproj file](csproj.md) or to a directory. If not specified, it defaults to the current directory.
```

Comment:
A couple of comments:
- .NET Core supports C#, F#, and Visual Basic, not just C#. 
- A path to a project file or to a directory can be interpreted as being the same thing.
The following is somewhat clearer: The argument is either the path and filename of a C#, F#, or Visual Basic project file, or the path to a directory that contains a C#, F#, or Visual Basic project file.

Reply:
Thanks for adding further clarification to this.  I'm happy to go with your contribution.